src/sagemaker/session.py

```diff
@@ -4284,9 +4285,12 @@ def create_endpoint(self, endpoint_name, config_name, tags=None, wait=True, live
             tags, "{}.{}.{}".format(SAGEMAKER, ENDPOINT, TAGS)
         )
 
-        self.sagemaker_client.create_endpoint(
+        res = self.sagemaker_client.create_endpoint(
             EndpointName=endpoint_name, EndpointConfigName=config_name, Tags=tags
         )
+        if res:
```

Comment:
CreateEndpoint in LocalMode does not return response
https://github.com/aws/sagemaker-python-sdk/blob/e75865ff02dcb3f5dc5d940a831843f0da8a76f6/src/sagemaker/local/local_session.py#L363-L381